src/components/atoms/Icon/Icons/OpenPolicyAgent.tsx

```diff
@@ -0,0 +1,25 @@
+const OpenPolicyAgent = () => (
```

Comment:
We seem to use both. Any reason why sometimes we use asset and sometimes in-code? The latter provides the	<Icon /> wrapper with a nice API which avoids needing more styled component boilerplate.

Let me know which of the two methods you want to double down on and I'll make it consistent.

Reply:
I think we are inconsistent because we didn't set a general way of doing it from the start. You can keep it like this for now and I can open an issue to make these consistent. I like having them as a React component to give us flexibility.